cmd/olclient/gov.go

```diff
@@ -197,10 +197,16 @@ func listProposals(cmd *cobra.Command, args []string) error {
 
 	// List single proposal
 	if listArgs.ProposalID != "" {
-		req := client.ListProposalRequest{
-			ProposalId: governance.ProposalID(listArgs.ProposalID),
+		pID := governance.ProposalID(listArgs.ProposalID)
+		if listArgs.ProposalID != "" {
```

Comment:
why there is duplicate of `if listArgs.ProposalID != ""`?
and inside pID.Err() we are still checking if proposal id is empty

Reply:
> why there is duplicate of `if listArgs.ProposalID != ""`? (line 199 and line 201)
> and inside pID.Err() we are still checking if proposal id is empty

Correct. I'll remove this unnecessary check.